util/ckan_loader.py

```diff
@@ -150,4 +150,4 @@ def load_data(adr_url, apikey):
 
 
 if __name__ == '__main__':
-    load_data('http://ckan:5000', "6011357f-a7f8-4367-a47d-8c2ab8059520")
+    load_data('http://ckan', "6011357f-a7f8-4367-a47d-8c2ab8059520")
```

Comment:
This is only a hint on how to use the module. You could even consider it a piece of doc.
If we want to change it it should say `http://dev-adr` IMO.